content/rooms/fragments/v2-state-res.md

```diff
@@ -112,9 +121,9 @@ the auth event is not rejected.
 
 The *resolution* of a set of states is obtained as follows:
 
-1.  Take all *power events* and any events in their auth chains,
-    recursively, that appear in the *full conflicted set* and order them
-    by the *reverse topological power ordering*.
+1.  Select all *power events* that appear in the *full conflicted set*. Compute
+    the union of their auth chains. Sort the union using the
+    *reverse topological power ordering*.
```

Comment:
currently it's unclear if the union to be sorted includes the power events themselves, or just their auth chains.

```suggestion
1.  Select all *power events* that appear in the *full conflicted set*. Compute
    the union of their auth chains, including the power events themselves.
    Sort the union using the *reverse topological power ordering*.
```

Reply:
I think when I was writing this I was thinking that the auth chain of E contains E? Can't remember if that's wrong; either way this is clearer

Reply:
b573c953. Not sure why this didn't apply before; I certainly meant it to.